### DISCUSSION TITLE
有app的网盘吗？

### DISCUSSION THREAD

**Discussion by yihangyingzi** [share121/inter-knot#4961] [委托]
app老是下一半下载失败，开了加速器也没有😭

**Comment by share121** [comment]
新版本我加入了加速源选项，可以选择一个加速源进行下载

**Reply by yihangyingzi** [reply]
👌🥰

**Comment by xiaqizhengzhao** [comment]
你可以试试电脑上下载完传到手机上

**Comment by xiaqizhengzhao** [comment]
我到作者的GitHub上下了一个传网盘了。https://www.123pan.com/s/v6nzVv-3cX0?提取码:xhVJ

**Reply by yihangyingzi** [reply]
好耶！

**Comment by xiaqizhengzhao** [comment]
校验值sha-256：38ceeee025b442e1753d2a2a53ac08076857ea285d52d6045eb6e59f2d4ef0d3

**Comment by Carkys** [comment]
我刚才电脑上下的也挺慢100kB左右，后来用Motrix下的1.5MB左右
